src/main/java/fr/cnes/sonarqube/plugins/framac/sensor/FramaCSensor.java

```diff
@@ -41,6 +46,7 @@
      */
     private static final Logger LOGGER = Loggers.get(FramaCSensor.class);
 
+    private static final Charset ENCODING = StandardCharsets.UTF_8;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "ENCODING" private field. ([squid:S1068](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonar-frama-c-plugin&pullRequest=23&issues=AWY0CuRBvOWp7Yd6eg9o&open=AWY0CuRBvOWp7Yd6eg9o)

[ISSUE_KEY]: AWY0CuRBvOWp7Yd6eg9o